test/cluster-tests/dcos/test.sh

```diff
@@ -64,6 +68,8 @@ while [[ ${count} -lt 25 ]]; do
   if [[ "${running}" == "3" ]]; then
     log "Found 3 running tasks"
     break
+  else
+    ${remote_exec} ./dcos marathon app list
```

Comment:
This has been put to get a status of app in case of failure. So right now when validation fails, there is no way to check what the current app status looks like